### DISCUSSION TITLE
Create a floating window that can be used alongside the page.

### DISCUSSION THREAD

**Discussion by avin-kavish** [radix-ui/primitives#2427] [Help] [answered]
The default dialog implementation closes when a user clicks outside the window. I want a dialog/floating window I can use alongside other content of the page and doesn't close when anything outside it is clicked.

Is there a way to implement this currently?

**Comment by benoitgrelard** [comment, marked as answer]
Yes, you are looking for a [non-modal dialog](https://www.radix-ui.com/primitives/docs/components/dialog#root).
<img width="409" alt="CleanShot 2024-02-27 at 14 59 56@2x" src="https://github.com/radix-ui/primitives/assets/1539897/d4fd6728-3c85-4352-a469-ddff5c62ea36">
